services/order.py

```diff
@@ -0,0 +1,39 @@
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from datetime import datetime
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(
+    tickets: list[dict],
+    username: datetime,
+    date: datetime | None = None
+) -> None:
+
+    with transaction.atomic():
```

Comment:
You could use `transaction.atomic` as a function decorator to cover all operations with a function here and in `create_movie`, prefer using it as a context manager in case you need more granular control.